app/views/pages/index.html.erb

```diff
@@ -11,7 +11,25 @@
       </form>
       <form class="navbar-form navbar-left" role="search">
         <div class="form-group">
-          <input type="text" class="form-control" placeholder="District Name">
+          <select class="form-control">
```

Comment:
The best way to having this is to define it in `application_helper.rb` a method that generates this:

```rb
def districts_options
  districts = ["Kennedy Town", "HKU", "Sai Ying Pun", "Sheung Wan", "Central", "Admiralty", "Wan Chai", "Causeway Bay", "Tin Hau", "Fortress Hill", "North Point", "Quarry Bay", "Tai Koo", "Sai Wan Ho", "Shau Kei Wan", "Heng Fa Chuen", "Chai Wan"]
 
 html = '<select class="form-control">'
 districts.each do |district|
  html += "<option>#{district}</option>"
  end
 html += "</select>""

 html
end
```